editor/static/scripts/output.js

```diff
@@ -122,7 +122,12 @@ function register(myLayout) {
         });
 
         container.getElement().on("click", function () {
-            editor.focus();
+            setTimeout(() => {
+                if (window.getSelection().rangeCount == 0 ||
+                    window.getSelection().getRangeAt(0).collapsed) {
+                    editor.focus();
+                }
+            }, 100);
```

Comment:
This is probably a dumb question, but I don't understand why the timeout has to be `100` rather than just `0` - when I try setting it to `0`, it works for me on a couple browsers.

Reply:
Yeah, I guess 0 does work here.